ntoskrnl/config/cmlazy.c

```diff
@@ -9,7 +9,7 @@
 /* INCLUDES *******************************************************************/
 
 #include "ntoskrnl.h"
-#define NDEBUG
+//#define NDEBUG
```

Comment:
intended? Or leftover?

Reply:
Intended, as it enables verbose debugging of the lazy flusher since it's now in action after a couple of years not doing anything so it's best to examine the behavior of the lazy flusher as ROS matures with time, until it'll be disabled back.